CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [Audit] Logging user actions is now opt-in (PR [#5151](https://github.com/vatesfr/xen-orchestra/pull/5151))
+- [Audit] Show warning in case of disabled user actions logging (PR [#5152](https://github.com/vatesfr/xen-orchestra/pull/5152))
```

Comment:
```suggestion
- [Settings/Audit] Warn if logging is not active (PR [#5152](https://github.com/vatesfr/xen-orchestra/pull/5152))
```

Reply:
Why `settings` ?

Reply:
Because this change does not apply to the plugin specifically but to the *Settings/Audit* page of xo-web.